edudoc/judge-tutorial/judge-tutorial.md

```diff
@@ -0,0 +1,83 @@
+# Judging Tutorial for Newcomers
+
+This document is intended for those who have not been a judge before, and will highlight some of the key regulations to follow during the judging process. The procedure described is intended for regular speedsolving events. For blindfolded events there are a few key differences not highlighted in this document.
+
+## Equipment
+
+This is the main equipment and tools required to proceed with a speedsolving attempt:
+
+**Stackmat timer**: This is the official device for measuring times at a WCA competition. The image below is the latest version of the timer, previous versions of the timer are still accepted to be used in competitions.
+
+![](images/timer.png){.centered}
+
+**Stopwatch**: This is used for measuring the inspection time. Stopwatches are also used to measure time for attempts over 10 minutes. **Note: Not to be confused with a stackmat timer.**
+
+## Judging Procedure
+
+1. Place the scrambled puzzle on the mat and keep it covered before the start of every attempt.
+2. Reset both the stackmat timer and stopwatch.
+3. The competitor has one minute to get ready when asked.
+4. Once the competitor is ready, uncover the puzzle and start the stopwatch simultaneously to time the inspection. This starts the attempt.
+5. After 8 seconds of inspection have elapsed you must call “8 seconds”. After 12 seconds of inspection have elapsed, you must call “12 seconds”.
+6. During the inspection phase, the competitor must not make any move to the puzzle, except to correct minor misalignments.
+7. The competitor starts the solve by placing their fingers on the sensors, waiting for the green light to turn on and then removing their hands from the timer.
+8. Stop the stopwatch as soon as the competitor starts their solve.
+9. The competitor stops the timer once they have solved the puzzle to stop the attempt. Inspect the puzzle without touching it (except for the Clock event) and call “OKAY”, “PENALTY”, or “DNF”.

```

Comment:
In roughly 8 years of competition experience, I have only noticed two occasions where judges call out the result loud.
1. If you're judging in a World Championship final where you're really adhering to the Regulations by the letter
2. If you're judging and your name happens to be Ron van Bruchem

Reply:
I strongly believe that we should still tell the judges to judge as stated in the regulations. Just because people don't do it, doesn't mean they shouldn't

Reply:
Okay